labelbox/__init__.py

```diff
@@ -2,6 +2,10 @@
 __version__ = "3.36.1"
 
 from labelbox.client import Client
+from labelbox.data import annotation_types as types
+from labelbox.data import metrics
```

Comment:
We don't really want customers using this now that we have auto metrics. I would just remove this (it also has the same dep issue)

Reply:
There are a couple of notebooks that use metrics:
* examples/model_diagnostics
* examples/integrations/detectron2
should we remove metrics calculations from these notebooks?

Reply:
You could lazy import or just in those notebooks (which are basically deprecated) keep the old import path.

Reply:
@maximeLabelbox I don't think we need any metrics libraries anymore because of auto-metrics. Can you confirm? 
Only place that will need metrics will be custom metrics, and we should probably create a separate notebook for that

Reply:
I have removed metrics from the root namespace, but the above notebooks still use metrics.